testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1571,8 +1589,9 @@
   raise "Folder exists" if folder_exists?($server, folder)
 end
 
-When(/^I ensure folder "(.*?)" doesn't exist$/) do |folder|
-  folder_delete($server, folder) if folder_exists?($server, folder)
+When(/^I ensure folder "(.*?)" doesn't exist on "(.*?)"$/) do |folder, host|
+  node = get_target(host)
+  folder_delete($server, folder) if folder_exists?(node, folder)
```

Comment:
does this always happen on the server or should it be
```suggestion
  folder_delete(node, folder) if folder_exists?(node, folder)
```